components/org.wso2.carbon.identity.application.authenticator.samlsso/src/test/resources/conf/application-authentication.xml

```diff
@@ -0,0 +1,192 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+<!--
+~ Copyright (c) 2014, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+~
+~ Licensed under the Apache License, Version 2.0 (the "License");
+~ you may not use this file except in compliance with the License.
+~ You may obtain a copy of the License at
```

Comment:
Just a suggestion. What if we use only elements which are needed for test in this file. Otherwise these files become stale with our changes into confs we do regularly. Isn't that a better approach ?

Reply:
Yes. that is a good approach. But for FileBasedConfigurationBuilder, we have to pass the full application-authentication.xml file. I will try to work on this future.